src/main/thrift/parquet.thrift

```diff
@@ -175,11 +175,6 @@ enum ConvertedType {
    */
   INTERVAL = 21;
 
-  /**
-   * Annotates a column that is always null
-   * Sometimes when discovering the schema of existing data
-   * values are always null
-   */
```

Comment:
Oops, I intended to remove NULL entirely because it is an unreleased type that can be replaced with the LogicalType version.